### PR TITLE
Add support for eventbridge scheduler

### PR DESCRIPTION
**Pull request by GrahamCampbell**
AWS have launched a new service, the Amazon EventBridge Scheduler (internally, named `Scheduler`).

* https://aws.amazon.com/blogs/compute/introducing-amazon-eventbridge-scheduler/
* https://docs.aws.amazon.com/scheduler/latest/APIReference/Welcome.html

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
I'm not sure why the generated output is getting changed, here?

**Comment by GrahamCampbell** [comment]
Also, I am experiencing an issue trying to generate some of the operations:

```
./generate Scheduler -vvv

 Select the operation to generate:
  [0] <all operation>
  [1] <new operation>
  [2] CreateScheduleGroup
  [3] DeleteScheduleGroup
  [4] GetScheduleGroup
  [5] ListScheduleGroups
 > 1

 Select the operation(s) to generate:
  [0] CreateSchedule
  [1] DeleteSchedule
  [2] GetSchedule
  [3] ListSchedules
  [4] ListTagsForResource
  [5] TagResource
  [6] UntagResource
  [7] UpdateSchedule
 > 0


In GeneratorHelper.php line 62:
                                                                        
  [RuntimeException]                                                    
  No camel case property "EnableECSManagedTags" is not yet implemented  
                                                                        

Exception trace:
  at /Users/graham/GitHub/async-aws/aws/src/CodeGenerator/src/Generator/GeneratorHelper.php:62
 AsyncAws\CodeGenerator\Generator\GeneratorHelper::normalizeName() at /Users/graham/GitHub/async-aws/aws/src/CodeGenerator/src/Generator/ObjectGenerator.php:201
 AsyncAws\CodeGenerator\Generator\ObjectGenerator->namedConstructor() at /Users/graham/GitHub/async-aws/aws/src/CodeGenerator/src/Generator/ObjectGenerator.php:90
 AsyncAws\CodeGenerator\Generator\ObjectGenerator->generate() at /Users/graham/GitHub/async-aws/aws/src/CodeGenerator/src/Generator/ObjectGenerator.php:179
 AsyncAws\CodeGenerator\Generator\ObjectGenerator->namedConstructor() at /Users/graham/GitHub/async-aws/aws/src/CodeGenerator/src/Generator/ObjectGenerator.php:90
 AsyncAws\CodeGenerator\Generator\ObjectGenerator->generate() at /Users/graham/GitHub/async-aws/aws/src/CodeGenerator/src/Generator/InputGenerator.php:118
 AsyncAws\CodeGenerator\Generator\InputGenerator->generate() at /Users/graham/GitHub/async-aws/aws/src/CodeGenerator/src/Generator/OperationGenerator.php:104
 AsyncAws\CodeGenerator\Generator\OperationGenerator->generate() at /Users/graham/GitHub/async-aws/aws/src/CodeGenerator/src/Command/GenerateCommand.php:273
 AsyncAws\CodeGenerator\Command\GenerateCommand->generateService() at /Users/graham/GitHub/async-aws/aws/src/CodeGenerator/src/Command/GenerateCommand.php:151
 AsyncAws\CodeGenerator\Command\GenerateCommand->generateServicesSequential() at /Users/graham/GitHub/async-aws/aws/src/CodeGenerator/src/Command/GenerateCommand.php:88
 AsyncAws\CodeGenerator\Command\GenerateCommand->execute() at /Users/graham/GitHub/async-aws/aws/vendor/symfony/console/Command/Command.php:298
 Symfony\Component\Console\Command\Command->run() at /Users/graham/GitHub/async-aws/aws/vendor/symfony/console/Application.php:1040
 Symfony\Component\Console\Application->doRunCommand() at /Users/graham/GitHub/async-aws/aws/vendor/symfony/console/Application.php:301
 Symfony\Component\Console\Application->doRun() at /Users/graham/GitHub/async-aws/aws/vendor/symfony/console/Application.php:171
 Symfony\Component\Console\Application->run() at /Users/graham/GitHub/async-aws/aws/src/CodeGenerator/generate.php:21
 require() at /Users/graham/GitHub/async-aws/aws/generate:32

generate [--all] [-r|--raw] [--] [<service> [<operation>]]
```

**Comment by jderusse** [comment]
You need to update the list in `src/CodeGenerator/src/Generator/GeneratorHelper.php` to add an exception for `'ECS' => 'Ecs'`

**Comment by GrahamCampbell** [comment]
Any idea why the generated output tests are failing?

**Comment by GrahamCampbell** [comment]
Hmm, in both the integration and client tests, generation seems to delete all the other methods, on every new method gen.

File: composer.json [hunk 1]
```diff
@@ -123,6 +124,7 @@
             "AsyncAws\\Rekognition\\Tests\\": "src/Service/Rekognition/tests",
             "AsyncAws\\Route53\\Tests\\": "src/Service/Route53/tests",
             "AsyncAws\\S3\\Tests\\": "src/Service/S3/tests",
+            "AsyncAws\\Scheduler\\Tests\\": "src/Service/Schedulertests",
```
Comment (jderusse) [inline composer.json:127]: This is why tests are overriden by the generator

```suggestion
            "AsyncAws\\Scheduler\\Tests\\": "src/Service/Scheduler/tests",
```

File: docs/clients/scheduler.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+```php
+use AsyncAws\Scheduler\Input\CreateScheduleGroupInput;
+use AsyncAws\Scheduler\SchedulerClient;
+
+$scheduler = new SchedulerClient();
+
+$result = $scheduler->createScheduleGroup(new CreateScheduleGroupInput([
+    'ClientToken' => 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
+    'Name' => 'foo',
```
Comment (jderusse) [inline docs/clients/scheduler.md:20]: Could you please use "realistic" values (liske a real usecase of  the service) in the documentation? It helps to understand how it works

File: src/Integration/Symfony/Bundle/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 ## NOT RELEASED
 
+### Added
+
+- Support for Scheduler
```
Comment (jderusse) [inline src/Integration/Symfony/Bundle/CHANGELOG.md:7]:
```suggestion
- Support for Scheduler

```

**Comment by GrahamCampbell** [comment]
🎉

**Review by jderusse — APPROVED** [review]

**Comment by jderusse** [comment]
Thank you @GrahamCampbell

**Pull request merged**